app/src/main/res/layout/activity_main.xml

```diff
@@ -1,16 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
why did you remove tools 
but @jaindiv26 done this issue please pick any other issue